### PR TITLE
Some more CLI stubs into Smart Class Parameter

### PR DESCRIPTION
**Pull request by jyejare**
Few more stubs for sub-commands:
1. list
2. remove
3. value hiding

### CODE REVIEW

File: tests/foreman/cli/test_classparameters.py [hunk 1]
```diff
@@ -804,6 +882,25 @@ def test_negative_create_matcher_avoid_duplicate(self):
...
     @stubbed()
+    @tier2
+    def test_positive_remove_matcher(self):
+        """Removal of matcher from parameter.
+
+        @feature: Class Parameters - Matcher
+
+        @steps:
+
+        1.  Override the parameter and create a matcher
```
Comment (elyezer) [inline tests/foreman/cli/test_classparameters.py]: Nitpick: use just one space after the `N.`. Also is better to align as:

```rst
1. My first line
   My second line
```

Comment (elyezer) [inline tests/foreman/cli/test_classparameters.py]: Please make sure this is fixed for other lists too.

**Comment by elyezer** [comment]
ACK, pending nitpick.

PS.: I don't know much about this feature.

**Comment by sthirugn** [comment]
ACK pending @elyezer's comments

**Comment by jyejare** [comment]
@elyezer : Done for this PR! For whole file, I will raise a new PR. Thanks!

**Comment by elyezer** [comment]
ACK, I will let @sthirugn take a second look before merging.

**Pull request merged**